### PR TITLE
Fix issues with creating custom resources from yaml 

### PR DESCRIPTION
**Pull request by stonepreston**
Fixes issue #18

### CODE REVIEW

**Comment by addyess** [comment]
LGTM 👍🏻

File: lightkube/codecs.py [hunk 1]
```diff
@@ -35,7 +35,8 @@ def _load_model(version, kind):
     try:
         module = importlib.import_module(f'lightkube.resources.{version.lower()}')
     except ImportError as e:
-        raise LoadResourceError(str(e))
+        # It was not a k8s resource and a generic resource was not previously defined
+        raise LoadResourceError(f"{str(e)}. If using a CRD, ensure you define a generic resource.")
```
Comment (gtsystem) [inline lightkube/codecs.py:39]:
```suggestion
        raise LoadResourceError(f"{e}. If using a CRD, ensure you define a generic resource.")
```

File: tests/test_codecs.py [hunk 2]
```diff
@@ -202,4 +202,44 @@ def test_dump_all_yaml():
                 x: y
             name: xyz
     """).lstrip()
-    assert res == expected
\ No newline at end of file
+    assert res == expected
+
+
+def test_load_model():
```
[the remark this answers is not in the document]
Comment (stonepreston) [reply tests/test_codecs.py]: Newest commit has tests added to test_from_dict_not_found() and test_from_dict_generic_res(). Thanks

File: lightkube/codecs.py (hunk 1 above)
Comment (stonepreston) [reply lightkube/codecs.py]: good catch

**Review by gtsystem — APPROVED** [review]
Good work! And thanks for the fix

**Pull request merged**